app/views/catalog/ds.html.erb

```diff
@@ -26,7 +26,8 @@
                @obj.datastreams[params[:dsid]].respond_to?(:ng_xml) ||
                params[:dsid] == 'RELS-EXT'
              ) &&
-             params[:dsid] != 'workflows'
+             params[:dsid] != 'workflows' &&
```

Comment:
Can't this line be removed now as it's not in the list?

Reply:
Sure. dropped.